src/fuzzy_match.cc

```diff
@@ -350,7 +339,8 @@ namespace fuzzy
     return std::log(num_sentences);
   }
 
-  std::vector<float> FuzzyMatch::compute_idf_penalty(const std::vector<unsigned>& pattern_wids) const {
+  std::vector<float> FuzzyMatch::compute_idf_penalty(const std::vector<unsigned>& pattern_wids,
+                                                     float oov_penalty) const {
```

Comment:
I find `oov_penalty` not explicit.
What do you think about `vocab_unkown_word_penalty` ?